packit_service/worker/handlers/distgit.py

```diff
@@ -1323,11 +1323,13 @@ def run(self) -> TaskResults:
         for job in self.package_config.get_job_views():
             if job.type == JobType.koji_build and job.sidetag_group and job.dist_git_branches:
                 configured_branches = aliases.get_branches(
-                    *job.dist_git_branches,
-                    default_dg_branch="rawhide",
+                    *job.dist_git_branches, with_aliases=True
```

Comment:
it's not very clear to me from firstly reading the code, why do we use the default for `default_dg_branch` (`main`) here and later explicitly set `default_dg_branch="rawhide"` when getting the branches, could you please add a comment?

Reply:
yes, thanks!